hypothesis-python/src/hypothesis/internal/compat.py

```diff
@@ -51,7 +51,7 @@ def int_to_byte(i):
 
 
 def benchmark_time():
-    return time.monotonic()
+    return time.perf_counter()
```

Comment:
Nit: Do we still need this in compat? Perhaps we should just inline it at usage sites.